client/src/components/layout/PlanetList.js

```diff
@@ -2,44 +2,44 @@ import React, { useState, useEffect } from "react"
 import PlanetTile from "./PlanetTile.js"
 import NewPlanetForm from "./NewPlanetForm.js"
 
-const PlanetList = () => {
-  const [planets, setPlanets] = useState([])
-  const getPlanetsData = async () => {
-    try {
-      const response = await fetch("/api/v1/planets")
-      if (!response.ok) {
-        const newError = new Error("Error in the fetch!")
-        throw newError
-      }
-      const responseBody = await response.json()
-      setPlanets(responseBody.planets)
-    } catch (err) {
-      console.log(err)
+const PlanetList = (props) => {
+    const [planets, setPlanets] = useState([])
+    const getPlanetsData = async () => {
+        try {
+            const response = await fetch("/api/v1/planets")
+            if(!response.ok){
+                const newError = new Error("Error in the fetch!")
+                throw(newError)
+            }
+            const responseBody = await response.json()
+            setPlanets(responseBody.planets)
+        } catch(err) {
+            console.log(err)
+        }
     }
-  }
 
-  useEffect(() => {
-    getPlanetsData()
-  }, [])
+    useEffect(() => {
+        getPlanetsData()
+    }, [])
+   
+    const planetsArray = planets.map((planet) => {
+        return (
+                    <PlanetTile 
+                        key={planet.id} 
+                        id={planet.id}
+                        name={planet.name}
+                        description={planet.description}
+                        imageUrl={planet.imageUrl}
+                    />
```

Comment:
indentation is a little wack here. Make sure that all folks have updated their indentation. Please update
https://launchboston44.slack.com/archives/C06BVSXU754/p1714488607103099